pandas/core/arrays/timedeltas.py

```diff
@@ -288,6 +317,21 @@ def _evaluate_with_timedelta_like(self, other, op):
 
         return NotImplemented
 
+    __mul__ = _wrap_tdi_op(operator.mul)
+    __rmul__ = __mul__
+    __truediv__ = _wrap_tdi_op(operator.truediv)
+    __floordiv__ = _wrap_tdi_op(operator.floordiv)
+    __rfloordiv__ = _wrap_tdi_op(ops.rfloordiv)
+
+    if compat.PY2:
+        __div__ = __truediv__
```

Comment:
Is there an issue if we define `__div__` without the condition on `compat.PY2` (i.e. also define it for Python 3.x)?

Reply:
Not that I'm aware of, but it's the pattern we follow everywhere else